pkg/volume/local_volume/local_volume.go

```diff
@@ -201,6 +202,34 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 
+	// If fsGroup is not nil, we need to make sure the requested fsGroup doesn't
+	// conflict with fsGroup already set for previous pod.
+	if fsGroup != nil {
```

Comment:
There is one special case here.  If user creates a Pod with fsGroup, while the second Pod with no fsGroup; then this condition won't catch it. Is this a valid case?